frontend/src/components/shared/simple-snack-bar/SimpleSnackBar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Close as CloseIcon } from '@material-ui/icons';
+
+import './SimpleSnackBar.scss';
+import { Snackbar, IconButton } from '@material-ui/core';
+
+type SimpleSnackBarProps = {
+  message: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+}
+
+const SimpleSnackBar: React.FC<SimpleSnackBarProps> = (props: SimpleSnackBarProps) => {
+
+  return (
+    <React.Fragment>
+      <Snackbar anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }} open={props.open} autoHideDuration={3000}
```

Comment:
This is a general point of feedback - in my opinion, when you have to put props on multiple lines due to line-length concerns, it is more readable to have only one prop per line. As is, I need to scan a 2D array rather than a 1D array to find the prop I'm looking for. If you like this feedback, maybe consider adding to your linter?

Reply:
I set the linter to allow a max of two props on the same line, since some of these props are relatively small like "container" or "md".